### ISSUE TITLE
wandb python not syncing when tensboard logging to s3

### ISSUE THREAD

**Issue by lminer** [wandb/client#1461]
I'm only getting learning rate information logged when using wandb with tensorboard, when I'm logging to an s3 bucket. I'm not getting any of my metrics or losses.

I'm running an EC2 ubuntu 18.04 deep learning instance with TF 2.3.1 and wandb 0.10.8. Works fine when I log locally.

**Comment by tyomhak** [comment]
Hey @lminer , can you please update to latest wandb version, and check if the issue persists?

**Comment by lminer** [comment]
@tyomhak it's still there. I can't seem to spin up a tensorboard instance.

**Comment by vanpelt** [comment]
@lminer can you link me to a run that you logged with `sync_tensorboard=True` that doesn't have tensorboard available?

**Comment by lminer** [comment]
@vanpelt Here's one: https://wandb.ai/lminer/separate/runs/2qfyd3us?workspace=user-lminer

**Comment by vanpelt** [comment]
Hmm, it looks like it picked up some metrics but never synced the tfevent files.  It's still running so I would imagine it will eventually upload them.  Can you send the `debug-internal.log` in the wandb directory once it finishes to vanpelt@wandb.com?

**Comment by lminer** [comment]
Sure! Where is the log dumped to?

**Comment by vanpelt** [comment]
In the `wandb` directory relative to where you python file is.

**Comment by lminer** [comment]
@vanpelt just sent it to you via email

**Comment by sarapapi** [comment]
Hi, how did you solve it? 
I have the same problem here

**Comment by lminer** [comment]
Never solved it. @vanpelt did you ever get a chance to check this out?

**Comment by vanpelt** [comment]
Hey @lminer my colleague @KyleGoyette is looking into this this week.  If you could share some code or a colab that would be helpful.

Looking at the run you linked to I can see all metrics on the dashboard...  When syncing tensorboard being logged directly to S3 we don't save the tfevent file with us because it only exists in your bucket so we can't upload it to our cloud.

**Comment by lminer** [comment]
It's hard for me to share any code since this is for work. I didn't realize the issue with the tfevent file. Maybe the solution is to create a new `Tensorboard` callback that subclasses the tf.keras one and logs to two locations: s3 and a temporary local directory.

**Comment by lminer** [comment]
@vanpelt did you ever make any progress on this?

**Comment by vanpelt** [comment]
@lminer we made a few fixes in recent releases that should address this.  Can you try again with the most recent release?  If it still doesn't work can you send me a link to a failed run or a snippet of code and the version of tensorboard you're using to reproduce this?

**Comment by lminer** [comment]
@vanpelt Sure. I just started this [run ](https://wandb.ai/lminer/separate/runs/2saqv766?workspace=user-lminer)on the most recent version of wandb. I'm actually not getting any metrics at all anymore. Tensorboard is 2.4.0.

**Comment by vanpelt** [comment]
The code hasn't uploaded for that run yet, it likely will when it finishes.  Are you able to make a little snippet with dummy data that can reproduce the issue?

**Comment by lminer** [comment]
Sure thing. I just used the example from [here](https://github.com/wandb/client/blob/75d59dcb5359b58bd8731032a691a33eae8f4b17/standalone_tests/keras_tensorboard.py), adding in an s3 path for the tensorboard logdir.

```python
import wandb
import numpy as np
import time
import tensorflow as tf
import glob
import os

wandb.init(sync_tensorboard=True, resume=True)

wandb.config['nice'] = 'So cool fun'


class Logger(tf.keras.callbacks.Callback):
    def on_epoch_end(self, epoch, logs):
        time.sleep(2)
        wandb.log({"wild_metrics": logs, "interval": epoch * 10})


model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Conv2D(
    3, 3, activation="relu", input_shape=(28, 28, 1)))
model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(10, activation="softmax"))
model.compile(loss="sparse_categorical_crossentropy",
              optimizer="sgd", metrics=["accuracy"])

model.fit(np.ones((10, 28, 28, 1)), np.ones((10,)), epochs=17,
          validation_split=0.2, callbacks=[Logger(), tf.keras.callbacks.TensorBoard(log_dir="s3://path/to/your/tensorboard/logs")])
```

**Comment by vanpelt** [comment]
Thanks @lminer I was able to reproduce the issue and just pushed a fix.  You can try it now with: `pip install --upgrade git+git://github.com/wandb/client.git@bug/tensorboard_s3#egg=wandb`.  Let me know if you're still seeing issues, this will be included in the next release.

**Comment by lminer** [comment]
Thanks. It works!
